### PR TITLE
tidb: Clean up tidb test

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: tidb_test.go [hunk 1]
```diff
@@ -463,52 +463,36 @@ func (s *testSessionSuite) TestAutoincrementID(c *C) {
 	mustExecSQL(c, se, s.dropDBSQL)
 }
 
+func checkAutocommit(c *C, se Session, stmt string, isNil, expect bool) {
```
Comment (coocood) [inline tidb_test.go]: Here the expect value is compared with `DisableAutocommit`, so I think it's better to rename this method to `checkDisableAutocommit`.

**Comment by zimulala** [comment]
PTAL @coocood

**Comment by coocood** [comment]
LGTM

**Pull request merged**